introspection-engine/connectors/sql-introspection-connector/src/introspection/models.rs

```diff
@@ -8,6 +8,9 @@ pub(super) fn introspect_models(datamodel: &mut dml::Datamodel, ctx: &mut Contex
     let mut remapped_models = Vec::new();
     let mut remapped_fields = Vec::new();
     let mut reintrospected_id_names = Vec::new();
+    let mut models_without_identifiers = Vec::new();
+    let mut models_without_columns = Vec::new();
+    let mut unsupported_types = Vec::new();
```

Comment:
Not a fan of all these local variables for different types of warnings. These should probably live in the context, and the code that extracts them at the end of the function be abstracted.

Reply:
Would a comment in the code help? Just to give a hint to whoever is here next and goes "ick, I should figure out how to make this better".

Reply:
Good point, will write on.

Reply:
Done.